library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -374,6 +374,13 @@ impl Command {
                         return Err(io::Error::last_os_error());
                     }
                 }
+                #[cfg(target_os = "hurd")]
```

Comment:
Don't we have to rewrite the `not(target_os = "android")` to include `target_os = "hurd"`?

Reply:
No, the Hurd has *both* `SIGPIPE` and `SIGLOST` (the latter being hurd-specific), so we do want both blocks.